### ISSUE TITLE
Accessibility - Login / Search dialogs

### ISSUE THREAD

**Issue by mcorkum** [ravendesignsystem/rds#80]
Needs aria controls

**Comment by troychaplin** [comment]
@mcorkum what part of this needs aria controls? A specific element? When they are used as a part of an overlay? Can you please provide more information?

**Comment by mcorkum** [comment]
https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/dialog.html

The element clicked needs to have aria-haspopup="true" and aria-labelledby=[IDofElement]

When clicked, the hidden div should have aria-hidden=true and it should change to aria-hidden=true

**Comment by troychaplin** [comment]
Referenced in #77, keeping all masthead accessibility issues in one spot
